src/commands/geosearch.json

```diff
@@ -39,102 +39,112 @@
-                        "name": "longitude",
-                        "type": "double"
-                    },
-                    {
-                        "name": "latitude",
-                        "type": "double"
-                    }
-                ]
-            },
-            {
-                "name": "circle",
-                "type": "block",
-                "optional": true,
+                "name": "from",
+                "type": "oneof",
                 "arguments": [
                     {
-                        "token": "BYRADIUS",
-                        "name": "radius",
-                        "type": "double"
+                        "token": "FROMMEMBER",
+                        "name": "member",
+                        "type": "string"
                     },
                     {
-                        "name": "unit",
-                        "type": "oneof",
+                        "token": "FROMLONLAT",
+                        "name": "longitude_latitude",
+                        "type": "block",
                         "arguments": [
                             {
-                                "name": "m",
-                                "type": "pure-token",
-                                "token": "m"
+                                "name": "longitude",
+                                "type": "double"
                             },
                             {
-                                "name": "km",
-                                "type": "pure-token",
-                                "token": "km"
-                            },
-                            {
-                                "name": "ft",
-                                "type": "pure-token",
-                                "token": "ft"
-                            },
-                            {
-                                "name": "mi",
-                                "type": "pure-token",
-                                "token": "mi"
+                                "name": "latitude",
+                                "type": "double"
                             }
                         ]
                     }
                 ]
             },
             {
-                "name": "box",
-                "type": "block",
-                "optional": true,
+                "name": "by",
+                "type": "oneof",
                 "arguments": [
                     {
-                        "token": "BYBOX",
-                        "name": "width",
-                        "type": "double"
-                    },
-                    {
-                        "name": "height",
-                        "type": "double"
-                    },
-                    {
-                        "name": "unit",
-                        "type": "oneof",
+                        "name": "circle",
+                        "type": "block",
+                        "optional": true,
```

Comment:
again, since we're in `oneof`, do we need `optional`?

Reply:
Correct - resolved in 56613896e